docs/src/pages/careers/lead-designer.md

```diff
@@ -42,6 +42,7 @@ More importantly, our challenges go way beyond the ones of design systems.
 
 We envision a future where MUI becomes the default toolkit for web developers to create UIs.
 It's why we've been expanding our offering with Joy UI, MUI Base, and MUI Toolpad.
+
```

Comment:
I think that we should connect the two ideas together, the goal we refer to is the long term one (I assume).

```suggestion
```